packages/beacon-state-transition/test/perf/util/balance.test.ts

```diff
@@ -0,0 +1,18 @@
+import {itBench} from "@dapplion/benchmark";
+import {generatePerfTestCachedStatePhase0, perfStateId} from "../util";
+import {State} from "../types";
+import {getEffectiveBalances} from "../../../src/util";
+
+describe("getEffectiveBalances", () => {
+  itBench<State, State>({
+    id: `getEffectiveBalances - ${perfStateId}`,
```

Comment:
Use `runsFactor` option to scale down the 100 for loop iterations
https://github.com/dapplion/benchmark/blob/8f2fffda28262e08a446d8a8284d1d2235dc479f/src/types.ts#L40

Reply:
@dapplion I use `runsFactor: 100` 8 lines below, is that correct?

Reply:
Oh I'm blind, didn't see it hahaha. Yes it's correct :+1: